kubos/utils/git_common.py

```diff
@@ -38,3 +38,22 @@ def get_active_kubos_version():
     else:
         return None
 
+
+def checkout(tag, repo):
```

Comment:
I think we could give this a slightly more descriptive name, as it checks out a specific tag and then saves off that tag information.